### DISCUSSION TITLE
lolMiner 1.61

### DISCUSSION THREAD

**Discussion by Lolliedieb** [Lolliedieb/lolMiner-releases#1751] [General]
### Changes
- Slightly improved performance of Flux mining on Nvidia Ampere and Turing cards (by approx 1.5-2.5% compared to 1.60)
- Improved system memory footprint when mining Flux or Beam on Nvidia GPUs. This allows running the solvers on more then 8 GPUs with only 4G system memory. 
- Speed exposed by API is now smoothed, so especially on all Equihash based algorithms the hashrate figures derived from API values will appear more smooth

### Fixes
- Fixed a bug causing Flux mining on Nvidia Pascal GPUs not working
- Fixed a bug causing stale shares being counted as defect shares in API - and not correctly exposed in miner UI as well.

<hr /><em>This discussion was created from the release <a href='https://github.com/Lolliedieb/lolMiner-releases/releases/tag/1.61'>lolMiner 1.61</a>.</em>

**Comment by Nctwitch** [comment]
Pascal cards are working again, but hashrate is about half of what it was.

**Comment by Lolliedieb** [comment]
Can you confirm there is a card that did work in 1.59 and 1.61 where the
hashrate did half between the versions?

If yes, please share the card and its settings - we may have a code slower
then other mining software, but having a slowdown between own versions
would be a serious issue, cause it would show we took a wrong development
direction... (that we did not see coming on our testing cards)

Nctwitch ***@***.***> schrieb am Do., 13. Okt. 2022, 14:13:

**Reply by Nctwitch** [reply]
Ok well this is entirely my bad, the cards do work on 1.61. I was used to seeing my 1080's at around 43 sols with miniz, and when i saw between 24-25 i jumped the gun and didnt think about it. But yes same hashrate between 1.59 and 1.61.
